### ISSUE TITLE
iOS Widget

### ISSUE THREAD

**Issue by dandrzejewski** [Clon1998/mobileraker#141]
**Describe the solution you'd like**
I would like an iOS widget that contains information about my printer and the current print.  Progress percentage, ETA, and perhaps a frame from the camera.

**Comment by thatbluflame** [comment]
Agreed, or even the new live notifications like octopod has, just switched over from octopod and I already miss them.

**Comment by Clon1998** [comment]
> Agreed, or even the new live notifications like octopod has, just switched over from octopod and I already miss them.

I agree, but since I am using flutter for cross platform compatibility I rely on a 3rd party plugin for notifications.
Afaik, the dev is working on support for live notifications the last time i chatted with him.
But there is only so much I can do  as I am the only one actively developing mobileraker in my spare time😉

**Comment by thatbluflame** [comment]
> > Agreed, or even the new live notifications like octopod has, just switched over from octopod and I already miss them.
> 
> I agree, but since I am using flutter for cross platform compatibility I rely on a 3rd party plugin for notifications. Afaik, the dev is working on support for live notifications the last time i chatted with him. But there is only so much I can do as I am the only one actively developing mobileraker in my spare time😉

Ah ok, thank you, app is still fantastic even without it!

**Comment by hauserkristof** [comment]
Hi @Clon1998 !

Do you have any update on this?


Thanks,
Kristóf

**Comment by Clon1998** [comment]
> Hi @Clon1998 !
> 
> Do you have any update on this?
> 
> Thanks, Kristóf

All I can update you on is that I added widgets for Android and Ios to my to-do list and also found some potential packages that can help me create them.
